src/DGtal/geometry/curves/estimation/LambdaMST2D.ih

```diff
@@ -0,0 +1,117 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+/**
+ * @file LambdaMST2D.ih
+ * @author Kacper Pluta (\c kacper.pluta@esiee.fr )
+ * Laboratoire d'Informatique Gaspard-Monge - LIGM, France
+ *
+ * @date 2014/10/03
+ *
+ * This file is part of the DGtal library.
+ */
+
+template < typename TSpace, typename TSegmentation, typename Functor >
+inline
+DGtal::LambdaMST2DEstimator< TSpace, TSegmentation, Functor >::LambdaMST2DEstimator() : dssSegments ( 0 ) {}
+
+template < typename TSpace, typename TSegmentation, typename Functor >
+inline
+void
+DGtal::LambdaMST2DEstimator< TSpace, TSegmentation, Functor >::init ( const ConstIterator & itb, const ConstIterator & ite )
+{
+  myBegin = itb;
+  myEnd = ite;
+}
+
+template < typename TSpace, typename TSegmentation, typename Functor >
+inline
+void
+DGtal::LambdaMST2DEstimator< TSpace, TSegmentation, Functor >::attach ( const TSegmentation & segmentComputer )
```

Comment:
Perhaps use ConstAlias to specify that the lifetime of segmentComputer must exceed the lifetime of this method.
Juste write: ... ::attach( ConstAlias<TSegmentComputer> segmentComputer )
instead